worker/games.py

```diff
@@ -310,6 +310,14 @@ def parse_cutechess_output(p, remote, result, spsa, spsa_tuning, games_to_play,
       kill_process(p)
       return { 'task_alive': True }
 
+    # Parse line like this:
+    # Warning: New-eb6a21875e doesn't have option ThreatBySafePawn
+    if line.split()[0]=='Warning:':
+      message = r'Cutechess-cli says: "%s"' % line.strip()
+      result['message']=message
+      send_api_post_request(remote + '/api/stop_run', result).json()
```

Comment:
`.json()` is not needed

Reply:
Thanks. This was a left over of a test print. I will delete it of course.